src/pybind/mgr/rbd_support/perf.py

```diff
@@ -38,24 +39,53 @@
 REPORT_MAX_RESULTS = 64
 
 
+# {(pool_id, namespace)...}
+ResolveImageNamesT = Set[Tuple[int, str]]
+
+PoolKeyT = Tuple[str, str]
+
+# (time, [value,...])
+PerfCounterT = Tuple[int, List[int]]
+RawImageCounterT = Union[Tuple[None, None], Tuple[PerfCounterT, PerfCounterT]]
```

Comment:
`Tuple[None, None]` looks odd.

Reply:
yeah, it is a little bit odd this way. i wanted apply minimal change to the code for adding type annotations.

will create another commit to change this to to `Optional[Tuple[PerfCounterT, PerfCounterT]]`.

Reply:
maybe `Tuple[Any, Any]`?

Reply:
I think it defeats the purpose of adding typing annotation. I'd prefer a more strict typing if possible.

Reply:
done.